tests/starlark/load-library/fn-config.yaml

```diff
@@ -18,3 +18,4 @@ source: |
   load('time.star', 'time')
   load('xlsx.star', 'xlsx')
   load('zipfile.star', 'ZipFile')
+  load('krmfn.star', 'krmfn')
```

Comment:
I'd suggest adding an simple example using it.

Reply:
added!